contents/handbook/engineering/enterprise-prioritization.md

```diff
@@ -4,6 +4,8 @@ sidebar: Handbook
 showTitle: true
 ---
 
+> We use the following [GitHub Board](https://github.com/orgs/PostHog/projects/16) to track enterprise features.
```

Comment:
Should we consistently use "Scale" instead of "Enterprise"?

Reply:
Well I'm the wrong person to answer this. Is "Scale" now used to encompass what we previously understood as "Enterprise" features? My only concern is that if this introduces confusion, as "Scale" might not intuitively represent the types of customers we're targeting.

Reply:
Yeah I think it should be "Scale" based on the website https://posthog.com/pricing

cc @jamesefhawkins for confirmation though.

Reply:
Renamed 😉